recipe/meta.yaml

```diff
@@ -1,35 +1,35 @@
 {% set name = "category_encoders" %}
-{% set version = "2.6.0" %}
-{% set sha256 = "bd9cfe8430b6cc2d67eaa51000937983422ebd81f88d770c3dcbe6cf9563e96e" %}
+{% set version = "2.6.1" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   - url: https://github.com/scikit-learn-contrib/{{ name }}/archive/{{ version }}.tar.gz
-    sha256: {{ sha256 }}
+    sha256: 51d4c594cd102a3d30a4a00f3ffa62f73b3d2f4298e7a57df8a8046a8d7881f5
 
 build:
   number: 0
-  skip: true  # [py<37]
-  script:
-    - {{ PYTHON }} -m pip install --no-deps -vvv .
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - python
     - pip
     - wheel
+    - numpy >=1.14.0
     - setuptools
   run:
     - python
     - numpy >=1.14.0
     - scikit-learn >=0.20.0
     - scipy >=1.0.0
-    - statsmodels >=0.9.0
     - pandas >=1.0.5
     - patsy >=0.5.1
+    - statsmodels >=0.9.0
+    - importlib_resources  # [py<39]
```

Comment:
```suggestion
```
I don't see importlib_resrouces here: https://github.com/scikit-learn-contrib/category_encoders/blob/2.6.1/setup.py#L34

Reply:
Thanks! I think I was looking at the master branch rather than 2.6.1!